v19.1/encryption.md

```diff
@@ -191,7 +223,11 @@ Information about keys is written to [the logs](debug-and-error-logs.html), incl
 * Active/old key information at startup.
 * New key information after data key rotation.
 
-#### Changing encryption algorithm or keys
+### Changing encryption algorithm or keys
```

Comment:
While this is true at the highest level, I wonder if we could break these situations into something more obvious to users like breaking things out into more sections by including "Enable Encryption" and "Disable Encryption".